python/src/deltachat/direct_imap.py

```diff
@@ -27,10 +29,19 @@ def dc_account_extra_configure(account):
     """
     if not hasattr(account, "direct_imap"):
         imap = DirectImap(account)
-        if imap.select_config_folder("mvbox"):
+        for folder in imap.list_folders():
+            assert imap.select_folder(folder)
             imap.delete(ALL, expunge=True)
-        assert imap.select_config_folder("inbox")
-        imap.delete(ALL, expunge=True)
+
+        for f in MANUALLY_CREATED_FOLDER_NAMES:
```

Comment:
Since `list_folders` is done right above, maybe remove everything but inbox to avoid hardcoded list?